UnityProject/Assets/Scripts/UI/Core/Windows/TeleportWindow/TeleportButtonSearchBar.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace UI.Core.Windows
+{
+	public class TeleportButtonSearchBar : MonoBehaviour
+	{
+		private InputField Searchtext;
+
+		private List<GameObject> HiddenButtons = new List<GameObject>();
```

Comment:
Codacy found an issue: [Make 'HiddenButtons' 'readonly'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6316546)

Reply:
I wonder why it thinks this should be readonly. Where do you set the values for this one?

Reply:
I think it thinks that it needs to be read-only because you set the class once and then forget about it and that's in initialisation time